src/genesis/helpers/field_enums.py

```diff
@@ -86,9 +89,8 @@ class EventFields(NamedFields):
     block_id = 2
     type = 3
 
-    @classmethod
```

Comment:
removal of `classmethod`s for:
`error: Only instance methods can be decorated with @property  [misc]`